cookbooks/bcpc-hadoop/recipes/phoenix_query_server.rb

```diff
@@ -0,0 +1,42 @@
+# vim: tabstop=2:shiftwidth=2:softtabstop=2
+
+include_recipe 'bcpc-hadoop::phoenixqs_kerberos'
+
+qs_runas = node['bcpc']['hadoop']['phoenix']['phoenixqs']['username']
+
+user qs_runas do
+  comment 'Runs phoenix queryserver'
+  only_if { node['bcpc']['hadoop']['phoenix']['phoenixqs']['localuser'] }
+  # apperently these do not happen in order
+  # and group 'hadoop' was being run furst
+  notifies :modify, "group[add #{qs_runas} to hadoop]", :immediate
+end
+
+group "add #{qs_runas} to hadoop" do
+  group_name 'hadoop'
+  members [ qs_runas ]
+  append true
+  action :nothing
+  only_if { node['bcpc']['hadoop']['phoenix']['phoenixqs']['localuser'] }
+end
+
+configure_kerberos 'phoenixqs_keytab' do
```

Comment:
The ``qs_runas`` user is created and added to ``hadoop`` group when the attribute ``node['bcpc']['hadoop']['phoenix']['phoenixqs']['localuser']`` is set to ``true``. Do we need to perform the same condition checking here?

Reply:
All this does is kinit

Reply:
Sorry the comment was meant for the next step.